gitium/inc/class-gitium-menu.php

```diff
@@ -66,4 +77,15 @@ public function show_message() {
 			<?php endif;
 		}
 	}
+
+	protected function show_disconnect_repository_button() {
+		?>
+		<form name="gitium_form_disconnect" id="gitium_form_disconnect" action="" method="POST">
+			<?php
+				wp_nonce_field( 'gitium-admin' );
+		  ?>
+			<input type="submit" name="GitiumSubmitDisconnectRepository" value='<?php _e( 'Disconnect from repo', 'gitium' ); ?>' class="button secondary" onclick="return confirm('<?php _e( 'Are you sure you want to disconnect from the remote repository?', 'gitium' ); ?>')"/>&nbsp;
```

Comment:
I don't really know what the coding style says about dropping out of php mode and back in again.
I personally find it a bit hard to read. @Marius786 do you have any thoughts about this?